### PR TITLE
Additional translations and fix to publish button.

### PR DESCRIPTION
**Pull request by Shinsina**
Screen Shot 2021-05-26 at 12 21 41 PM
Screen Shot 2021-05-26 at 12 21 46 PM
Screen Shot 2021-05-26 at 12 21 51 PM

### CODE REVIEW

**Comment by Shinsina** [comment]
[How to solve for Mundo Leaders entries in Spanish and English](https://trello.com/c/oKzezqPm/50-how-to-solve-for-mundo-leaders-entries-in-spanish-and-english)

**Review by brandonbk — APPROVED** [review]

File: services/app/app/helpers/translations.js [hunk 1]
```diff
@@ -64,6 +64,7 @@ export default {
     logoLabel: 'Logo',
     uploadLabel: 'Upload new image',
     saveChangesLabel: 'Save Changes',
+    textLinkLabel: "Text link",
```
Comment (solocommand) [inline services/app/app/helpers/translations.js:67]: This should read `Link Text` not `Text link`. It is representative of the text value that should be included inside the A tag within the datacard -- "Cable and Hose Carrier Systems" in this example: 
image

There should be a `Link URL` label to be added as well.

Comment (solocommand) [inline services/app/app/helpers/translations.js:67]: Keep quotes consistant -- use single quotes in this file.

File: services/app/app/helpers/translations.js [hunk 2]
```diff
@@ -73,6 +74,11 @@ export default {
     titleLabel: 'Title',
     addContactLabel: 'Add a contact',
 
+    // Promotions
+    promotionsP1Label: 'Review your companys product promotions',
```
Comment (solocommand) [inline services/app/app/helpers/translations.js:78]: This should be `your company's product`

**Review by solocommand — CHANGES_REQUESTED** [review]

**Review by solocommand — APPROVED** [review]

**Pull request merged**